edi_sending_warning/models/edi_exchange_record.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+
+
+class EdiExchangeRecordListener(Component):
+    _name = "edi.exchange.listener"
+    _inherit = ["base.connector.listener"]
```

Comment:
Try this:
```suggestion
    _inherit = ["base.event.listener"]
```

Because this one is declared in module "component_event" (which is installed as dep of "edi_oca".

The other one (base.connector.listener) is part of "connector" module, which is not required.